src/libs/actions/IOU.ts

```diff
@@ -1698,7 +1700,8 @@ function getUpdateMoneyRequestParams(
         });
     }
 
-    if (policy && PolicyUtils.isPaidGroupPolicy(policy) && updatedTransaction) {
+    const isPartialTransaction = TransactionUtils.isMerchantMissing(updatedTransaction) && TransactionUtils.isAmountMissing(updatedTransaction);
+    if (policy && PolicyUtils.isPaidGroupPolicy(policy) && updatedTransaction && !isPartialTransaction) {
```

Comment:
@eh2077 There are 3 active usages of `getViolationsOnyxData` and the only one left that I don't handle yet is in `buildOnyxDataForTrackExpense`. There is no option to track expenses in WS, so we can't test it yet. Do we want to apply the same fix for track expenses? If yes, I think I will move the partial transaction check inside `getViolationsOnyxData` and just return null if it's a partial one.

cc: @cead22

Reply:
Good point 👍. I think we can apply the same fix for track expense as well. I find this issue https://github.com/Expensify/App/issues/38971 is going to add track expense to WS.

Reply:
Updated